00-Introducing-HOOMD-blue/05-Compressing-the-System.ipynb

```diff
@@ -0,0 +1,442 @@
+{
```

Comment:
<p>We should avoid fluid representations when teaching, so we should avoid covering multiple ways to do the same thing. We should also minimize the number of new concepts introduced at one time.</p><p> </p><p>In this case, the explicit names match the name of the concept the learner is working with and behave as close to a Python list as possible - presumably something they are already familiar with. </p><p> </p><p>The convenience syntax methods for <code>Operations</code> hide the connection to the type of object which I'd be fine with if we decided that was the primary vehicle that users should use. However, since the methods for operations (add, remove, +=, -=) are different from the Python list, these are new concepts that must be taught. Thus I chose the explicit route.</p>

Reply:
I agree with that concept. Then we just need to make sure this is documented. Plus it doesn't work currently (unintuitive behavior with Python properties).